### ISSUE TITLE
No preview window for generated Backlinks - additonal idea?

### ISSUE THREAD

**Issue by Fantasyfr** [ylc395/joplin-plugin-note-link-system#20]
Hi,

i don't get preview window for generated Backlinks.
By the way it is possible to add additional content to the backlinks like a piece of the article where this note is mention? 

Best regards
Frank

**Comment by ylc395** [comment]
> i don't get preview window for generated Backlinks.

Do you mean in backlinks panel?

**Comment by Fantasyfr** [comment]
yeah the backlinks panel, i didn't thought to unfold it. Now i see what you mean, but a preview window for the backlinks would be also a nice feature.

by the way thank you for the fix :-)
